docroot/themes/humsci/humsci_basic/theme-settings.php

```diff
@@ -138,4 +138,16 @@ function humsci_basic_form_system_theme_settings_alter(array &$form, FormStateIn
     '#default_value' => theme_get_setting('global_footer_variant_classname'),
   ];
 
+  // Animation Enhancement
+  $form['options_settings']['humsci_basic_animation_enhancement'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Animation Enhancements'),
+  ];
+
+  $form['options_settings']['humsci_basic_animation_enhancement']['animation_enhancement'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Use animation enhancements'),
+    '#default_value' => theme_get_setting('animation_enhancement'),
```

Comment:
can you also add a description that mentions how this is useful for accessibility in many cases, or something along those lines of why someone might want to disable animations.

`'#description' => t("This enables/disables animations...because...."),`